scripts/hijri-calendar/index.js

```diff
@@ -19,29 +19,33 @@ const moment = require('moment-hijri')
 const filename = path.resolve(__dirname, '../../src/internal/hijri-calendar.js')
 const out = {}
 
+const YEAR0 = 580
```

Comment:
why 580?

Reply:
1 Muharram 1389 is at 1969-03-19 in gregorian year

1969 - 1389 = 580